converter/src/main/java/gov/cms/qpp/conversion/validate/ReportingParametersActValidator.java

```diff
@@ -1,22 +1,22 @@
 package gov.cms.qpp.conversion.validate;
 
-
 import gov.cms.qpp.conversion.decode.ReportingParametersActDecoder;
 import gov.cms.qpp.conversion.model.Node;
 import gov.cms.qpp.conversion.model.TemplateId;
 import gov.cms.qpp.conversion.model.Validator;
+import gov.cms.qpp.conversion.model.error.ErrorCode;
 
 @Validator(TemplateId.REPORTING_PARAMETERS_ACT)
 public class ReportingParametersActValidator extends NodeValidator {
-	public static final String PERFORMANCE_YEAR = "Must have a performance year";
-	public static final String SINGLE_PERFORMANCE_START = "Must have one and only one performance start";
-	private static final String SINGLE_PERFORMANCE_END = "Must have one and only one performance end";
 
 	@Override
 	protected void internalValidateSingleNode(Node node) {
 		check(node)
-				.singleValue(SINGLE_PERFORMANCE_START, ReportingParametersActDecoder.PERFORMANCE_START)
-				.singleValue(SINGLE_PERFORMANCE_END, ReportingParametersActDecoder.PERFORMANCE_END)
-				.value(PERFORMANCE_YEAR, ReportingParametersActDecoder.PERFORMANCE_YEAR);
+				.singleValue(ErrorCode.REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_START, 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 135 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)